test/steps/queries/2_when.js

```diff
@@ -15,6 +15,29 @@
  */
 import { processQueryResult } from '../../../src/utils/helpers';
 
+export function theQueryInstanceSendsARequestAndTheLiskAPIInstanceReturnsTheSuccessResponse() {
+	const { liskAPIInstance, queryInstance } = this.test.ctx;
+	const sendRequestResult = { success: true };
+	liskAPIInstance.sendRequest.resolves(sendRequestResult);
+	this.test.ctx.sendRequestResult = sendRequestResult;
+	const returnValue = queryInstance.sendRequest('', {});
+	this.test.ctx.returnValue = returnValue;
+	return returnValue.catch(e => e);
```

Comment:
Some blank lines would be helpful to separate the stages of this step (and the one below).